lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/179d_ashrae_90_1_2007.Model.rb

```diff
@@ -726,50 +724,115 @@ def model_create_prm_any_baseline_building(user_model, building_type, climate_zo
 
       if baseline_179d
 
-        OpenStudio.logFree(OpenStudio::Info, 'openstudio.standards.Model', '*** Baseline Adjust Design Outdoor Air Flow Rate to Proposed Levels if lower ***')
+        OpenStudio.logFree(OpenStudio::Info, 'openstudio.standards.Model', '*** Baseline Adjust Minimum/Design Outdoor Air Flow Rate to Proposed Levels if different ***')
         total_oa_design_flow_rate = 0.0
 
-        model.getAirLoopHVACs.each do |air_loop_hvac|
-          next if air_loop_hvac.airLoopHVACOutdoorAirSystem.empty?
-
-          # total_dsoa = air_loop_hvac.thermalZones.sum { |zone| zone_dsoas[zone.nameString] }
-
-          oa_design_flow_rate = nil
-          sizing_system = air_loop_hvac.sizingSystem
-          oa_system = air_loop_hvac.airLoopHVACOutdoorAirSystem.get
-          controller_oa = oa_system.getControllerOutdoorAir
-          # controller_mv = controller_oa.controllerMechanicalVentilation
-
-          if sizing_system.designOutdoorAirFlowRate.is_initialized
-            oa_design_flow_rate = sizing_system.designOutdoorAirFlowRate.get
-          elsif controller_oa.autosizedMinimumOutdoorAirFlowRate.is_initialized
-            oa_design_flow_rate = controller_oa.autosizedMinimumOutdoorAirFlowRate.get
+        # Get minimum and design outdoor airflow rates
+        outdoor_airflow_rate_minimum_m_3_per_s_proposed = outdoor_airflow_rates_proposed[0]
+        outdoor_airflow_rate_design_m_3_per_s_proposed = outdoor_airflow_rates_proposed[1]
+        outdoor_airflow_rates_baseline = get_minimum_and_design_outdoor_airflow_rates(model, 'BASELINE')
+        outdoor_airflow_rate_minimum_m_3_per_s_baseline = outdoor_airflow_rates_baseline[0]
+        outdoor_airflow_rate_design_m_3_per_s_baseline = outdoor_airflow_rates_baseline[1]
```

Comment:
getting minimum/design outdoor airflow rates for both proposed/baseline. preparing for comparisons.